pkg/parser/epub/testdata/package/accessibility-epub2.opf

```diff
@@ -20,6 +20,9 @@
     <meta name="a11y:certifiedBy" content="Accessibility Testers Group"/>
     <meta name="a11y:certifierCredential" content="DAISY OK"/>
     <meta name="a11y:certifierReport" content="https://example.com/a11y-report/"/>
+
```

Comment:
Why this testdata package does not have the  `<meta property="a11y:exemption">eaa-disproportionate-burden</meta>`?